README.md

```diff
@@ -6,6 +6,9 @@ range of moving assets (such as vehicles) and visualize them on a live map. The
 applications use a mixture of technologies - Android, Firebase,
 Google Maps, GTFS (General Transit Feed Specification), and more.
 
+**Note:** This application implements asset tracking. For use in a production environment, you therefore need a [Google Maps APIs Premium Plan license](
```

Comment:
How about:

**Note:** This application implements asset tracking. To use in a production environment you need a Google Maps APIs Premium Plan license.